shared/constants/gas.js

```diff
@@ -29,3 +29,12 @@ export const GAS_RECOMMENDATIONS = {
   MEDIUM: 'medium',
   HIGH: 'high',
 };
+
+/**
+ * These represent the different edit modes presented in the UI
+ */
+export const EDIT_GAS_MODES = {
```

Comment:
[REQ] These are UI specific, they don't need to be in the shared folder. I'd recommend leaving these declared in the EditPopover as they primarily modify the behavior of this component and should likely only be imported when also importing EditPopover

Reply:
Pushing back on this slightly; `GAS_RECOMMENDATIONS` is also a shared constant which is only used in the UI.  I think having it here will be really helpful here.

Reply:
I didn't comment on the addition of `GAS_RECOMMENDATIONS` because we'll need that in the background now, too. We'll be using it when we set default gas estimations on incoming transactions from dapps. 

I understand the desire to place this here, but it is important that we keep the shared folder strict to its purpose of storing code shared by both `app/` and `ui/` folders to keep the module boundary line clearly defined.